packages/openapi-to-graphql/src/resolver_builder.ts

```diff
@@ -143,13 +145,16 @@ export function getResolver({
     })
 
     // Handle arguments provided by links
-    for (let paramName in argsFromLink) {
+    for (const paramName in argsFromLink) {
+      const saneParamName = Oas3Tools.sanitize(
+        paramName,
+        !data.options.simpleNames
+          ? Oas3Tools.CaseStyle.camelCase
+          : Oas3Tools.CaseStyle.simple
+      )
+
       let value = argsFromLink[paramName]
 
-      let paramNameWithoutLocation = paramName
-      if (paramName.indexOf('.') !== -1) {
```

Comment:
This didn't make any sense to me. Please let me know if you can make any sense from it. I have deleted it for now.

Reply:
I think this is in reference to the parameter names in link objects. This is what a typical link object looks like:

```
      "Author": {
        "$ref": "ThisIsHereAndFormattedThisWayBecauseOfAnIssueWithADependency",
        "operationRef": "Example API 3#/paths/~1authors~1{authorId}/get",
        "parameters": {
          "authorId": "$request.path.username"
        },
        "description": "Link between two different APIs"
      }
```

I don't see why we need to check for `.` and possibly split it.